fixtures/vitest-pool-workers-examples/queues/test/queue-consumer-integration-self.test.ts

```diff
@@ -24,9 +24,9 @@ it("consumes queue messages", async () => {
 	];
 	const result = await SELF.queue("queue", messages);
 	expect(result.outcome).toBe("ok");
-	expect(result.retryAll).toBe(false); // `true` if `batch.retryAll()` called
+	expect(result.retryBatch.retry).toBe(false); // `true` if `batch.retryBatch.retry()` called
```

Comment:
Is this a change to the API syntax or a behavioural change?

Reply:
this is the new `FetcherQueueResult` object shape

Reply:
thank you for pointing that out 🙏 . Fixed!